go.work

```diff
@@ -45,6 +45,7 @@ use (
 	./services/rabbitmq
 	./services/redis
 	./services/resourcemanager
+	./services/runcommand
```

Comment:
I don't think we need to add it to the go.work just yet. This could be done when we implement examples

Reply:
If I remove it from go.work, linter is complaining that module not listed in go.work. As it has no disadvantage, i would keep it for now in go.work.